tests/cli/bootprep/input/test_configuration.py

```diff
@@ -307,6 +338,16 @@ def test_product_version_property_not_present(self):
         """Test the product_version property when version is not in the layer data"""
         self.assertEqual(LATEST_VERSION_VALUE, self.branch_layer.product_version)
 
+    def test_product_version_jinja_template(self):
+        """Test the product_version property when it uses Jinja2 templating"""
+        # Have to double the literal brackets that make up the Jinja2 variable reference
+        self.version_layer_data['product']['version'] = f'{{{{{self.product_name}.version}}}}'
```

Comment:
Would this be more readable?
```suggestion
        self.version_layer_data['product']['version'] = '{{' + f'{self.product_name}.version' + '}}'
```

Reply:
Yes, I think that helps distinguish between the brackets that are part of the Jinja template and the ones that are used by the fstring